hack/install.sh

```diff
@@ -237,7 +266,7 @@ do_install() {
 			exit 0
 			;;
 
-		fedora|centos)
+		fedora|centos|oracleserver)
```

Comment:
oracleserver wont work as lsb_dist for download in the url needs to be oraclelinux